panel/src/panel/content.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @since 5.0.0
+ */
+export default (panel) => {
+	return {
+		/**
+		 * Returns all fields and their values that
+		 * have been changed but not yet published
+		 *
+		 * @returns {Object}
+		 */
+		get changed() {
```

Comment:
Reading through the other changed files, I lean towards `this.$content.changes` for this. I assume that you thought about `changed` & `published` and I really like this as well. But we also use `changes` as version id, so I think it would be more consistent.